keyboards/yushakobo/primer79/readme.md

```diff
@@ -0,0 +1,27 @@
+# Primer79
+
+![Primer79](https://i.imgur.com/n57Td4T.jpg)
```

Comment:
```suggestion
![Primer79](https://i.imgur.com/n57Td4Th.jpg)
```

Reply:
I have corrected the areas you pointed out and would appreciate your review.